docs/standard/io/file-path-formats.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "File path formats on Windows systems"
+ms.date: "04/25/2018"
+ms.prod: ".net"
+ms.technology: dotnet-standard
+ms.topic: "article"
+helpviewer_keywords: 
+  - "I/O, long paths"
+  - "long paths"
+  - "path formats, Windows"
+author: "rpetrusha"
+ms.author: "ronpet"
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+# File path formats on Windows systems
```

Comment:
Do you think this topic should cover how case is handled in Windows file paths? There are some surprising (at least to unix folks) results: Creating a file or directory uses the case of the path / file name. Renaming to fix case is interesting. And, case does not need to match for accessing an existing file.

Reply:
@BillWagner, I think that that's a really good point, and a really important one. I'll add a section on case.